carma_wm_ctrl/src/WMBroadcaster.cpp

```diff
@@ -571,9 +572,18 @@ void WMBroadcaster::addGeofence(std::shared_ptr<Geofence> gf_ptr)
   autoware_lanelet2_msgs::MapBin gf_msg;
   auto send_data = std::make_shared<carma_wm::TrafficControl>(carma_wm::TrafficControl(gf_ptr->id_, gf_ptr->update_list_, gf_ptr->remove_list_));
   carma_wm::toBinMsg(send_data, &gf_msg);
-  map_update_pub_(gf_msg);
   
-
+  if(route_flag_==true)
```

Comment:
rather than having an if/else statement could could just assign the flag directing to the message field since both are boolean with a 1-1 mapping.

Reply:
Done